### PR TITLE
Fixes remaining eslint errors

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: src/components/areaMenu/AposAreaMenu.vue [hunk 1]
```diff
@@ -12,21 +12,25 @@
             <dt>
               <button
                 :for="item.label" class="apos-area-menu__group-label"
-                v-if="item.items" tabindex="0" :id="menuId + '-trigger-' + index"
+                v-if="item.items" tabindex="0"
+                :id="menuId + '-trigger-' + index"
```
Comment (gregvanbrug) [inline src/components/areaMenu/AposAreaMenu.vue:16]: Should we be using template literals? Besides this line there are also instances on `17`, `40`, `41`, `48`, `54`, `55`, `65`, `66`.

File: src/components/areaMenu/AposAreaMenu.vue [hunk 2]
```diff
@@ -142,6 +146,7 @@ export default {
     },
 
     toggleGroup(index) {
+      console.log(index);
```
Comment (gregvanbrug) [inline src/components/areaMenu/AposAreaMenu.vue:149]: Stray log.

File: src/components/button/AposButton.vue [hunk 3]
```diff
@@ -27,23 +32,39 @@ export default {
...
+      default: null
+    },
+    type: {
+      type: String,
+      default: null
+    },
     iconOnly: Boolean,
     iconColor: {
       type: String,
       default: ''
```
Comment (gregvanbrug) [inline src/components/button/AposButton.vue:54]: Should this be `null`? Seems like we're `null`ing all other string defaults here.

File: src/components/contextMenu/AposContextMenuTip.vue [hunk 4]
```diff
@@ -1,20 +1,32 @@
 <template>
-  <svg :class="[alignmentModifier, originModifier]" class="apos-context-menu__tip" width="27px" height="13px" viewBox="0 0 27 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <!-- Disabling since the SVG is mostly not active vue template code. -->
```
Comment (gregvanbrug) [inline src/components/contextMenu/AposContextMenuTip.vue:2]: Should we add an ignore pattern for this so we don't have to manually add this in the future? Or should `svg`s live somewhere else and be imported?

File: src/components/loading/AposLoading.vue [hunk 5]
```diff
@@ -1,5 +1,7 @@
 <template>
   <div class="apos-loading">
+    <!-- Disabling since the SVG is mostly not active vue template code. -->
```
Comment (gregvanbrug) [inline src/components/loading/AposLoading.vue:3]: See prior `svg` comment.

File: src/components/loading/AposSpinner.vue [hunk 6]
```diff
@@ -1,5 +1,7 @@
 <template>
   <div class="apos-spinner">
+    <!-- Disabling since the SVG is mostly not active vue template code. -->
```
Comment (gregvanbrug) [inline src/components/loading/AposSpinner.vue:3]: See prior `svg` comment.

File: src/components/modal/AposModalLip.vue [hunk 7]
```diff
@@ -1,4 +1,6 @@
 <template>
+  <!-- Disabling since the SVG is mostly not active vue template code. -->
```
Comment (gregvanbrug) [inline src/components/modal/AposModalLip.vue:2]: See prior `svg` comment.

File: src/components/tagApplyMenu/AposTagApply.vue [hunk 8]
```diff
@@ -17,7 +21,10 @@
       </div>
       <transition name="fade">
         <ol v-if="searchTags.length && !creating" class="apos-apply-tag-menu__tags">
-          <li class="apos-apply-tag-menu__tag" v-for="tag in searchTags" :key="keyPrefix + '-' + tag.slug">
+          <li
+            class="apos-apply-tag-menu__tag" v-for="tag in searchTags"
+            :key="keyPrefix + '-' + tag.slug"
```
Comment (gregvanbrug) [inline src/components/tagApplyMenu/AposTagApply.vue:26]: See prior `template literal` comment.

File: src/components/tagApplyMenu/AposTagApply.vue [hunk 9]
```diff
@@ -39,7 +46,9 @@
               :disable-focus="!open"
             />
           </p>
-          <span class="apos-apply-tag-menu__empty-icon">🌾</span>
+          <span class="apos-apply-tag-menu__empty-icon">
+            🌾
```
Comment (gregvanbrug) [inline src/components/tagApplyMenu/AposTagApply.vue:50]: ? No lie, at first I thought this was a smudge on my screen...

File: src/components/areaMenu/AposAreaMenu.vue (hunk 1 above)
Comment (abea) [reply src/components/areaMenu/AposAreaMenu.vue:16]: I have no problem with that. That wasn't a linter error or a change in this PR, so I'd prefer to manage that in a separate one.

File: src/components/button/AposButton.vue (hunk 3 above)
Comment (abea) [reply src/components/button/AposButton.vue:54]: We're actually no longer using this as of this PR. I switched to `currentColor`. @stuartromanek should the icon ever be a different color from the text? If not we can simply delete this prop.

Comment (stuartromanek) [reply src/components/button/AposButton.vue:54]: looks like we're still using `iconColor` in `AposPiecesManagerToolbar` ?
It provides us with a way to 'activate' a button that would be dependent on the component's state

File: src/components/contextMenu/AposContextMenuTip.vue (hunk 4 above)
Comment (abea) [reply src/components/contextMenu/AposContextMenuTip.vue:2]: The trouble is that some of these SVGs do have reactive attributes, so I don't think we can store them elsewhere and import. There might be some regex way to do that. I'm not sure yet that it's worth the effort.

Comment (abea) [reply src/components/contextMenu/AposContextMenuTip.vue:2]: I'd be happy to be wrong there. I just don't want to fight with regex for this right now.

File: src/components/tagApplyMenu/AposTagApply.vue (hunk 9 above)
Comment (abea) [reply src/components/tagApplyMenu/AposTagApply.vue:50]: lol

File: src/components/contextMenu/AposContextMenuTip.vue (hunk 4 above)
Comment (gregvanbrug) [reply src/components/contextMenu/AposContextMenuTip.vue:2]: Should be a way to do that without a regex. I guess we can look at that later.

**Pull request merged**